client/starwhale/core/model/model.py

```diff
@@ -591,8 +613,55 @@ def buildImpl(self, workdir: Path, **kw: t.Any) -> None:  # type: ignore[overrid
             ),
             (self._make_auto_tags, 5, "make auto tags"),
         ]
+
         run_with_progress_bar("model bundle building...", operations)
 
+    def _package_runtime(self, runtime_uri: URI | str) -> None:
+        if isinstance(runtime_uri, str):
+            uri = URI.guess(runtime_uri, fallback_type=URIType.RUNTIME)
+        else:
+            uri = runtime_uri
+
+        # TODO: support runtime uri in the cloud instance
+        if uri.instance_type != InstanceType.STANDALONE:
+            raise NoSupportError(
+                f"runtime type {uri.instance_type} not support in package runtime"
+            )
+
+        info: t.Dict[str, t.Any] = {
+            "name": uri.object.name,
+        }
+
+        if uri.object.typ == URIType.RUNTIME:
+            runtime = StandaloneRuntime(uri)
+            bundle_path = runtime.store.bundle_path
+            info = {
+                "name": runtime.name,
+                "manifest": runtime.store.manifest,
+                "hash": blake2b_file(bundle_path),
+            }
+        elif uri.object.typ == URIType.MODEL:
+            model = StandaloneModel(uri)
+            bundle_path = model.store.packaged_runtime_bundle_path
+            packaged_runtime = model.store.manifest.get("packaged_runtime")
+            if not packaged_runtime or not bundle_path.exists():
+                raise RuntimeError(f"model {uri} not packaged runtime")
+
+            info = packaged_runtime
+        else:
+            raise NoSupportError(
+                f"uri type {uri.object.typ} not support to package runtime"
+            )
+
+        dest = self.store.packaged_runtime_bundle_path
+        console.print(f":optical_disk: package runtime({uri}) to {dest}")
+        copy_file(bundle_path, dest)
```

Comment:
should we ignore the venv/conda which in the origin dir, because copy it to ta new dir will not work in the new path and use many space

Reply:
bundle_path is a swrt file, which is a tar of runtime